projects/worker-basic/.npmignore

```diff
@@ -0,0 +1,25 @@
+.vscode
```

Comment:
I don't think we will ever `npm publish` these, so we should drop this file.

BTW, same for the generator in case it currently adds this one. @tmathern